### ISSUE TITLE
Unable to add MeSH Geographic Headings

### ISSUE THREAD

**Issue by bushmanb** [LD4P/qa_server#487]
For tracking purposes, please create a separate issue for each indexing improvement request.

- [ ] Please describe the expected performance and accuracy of a particular vocabulary. 

When using the Lookup for Subject of the Work on the PCC BF2 Work (Monograph), we are unable to search and add MeSH geographic subjects (e.g. United States, California, etc.)  I also noticed that it didn't work for LCSH either.  Geographic subjects only seems to available in the separate FAST vocabulary.

- [ ] If applicable, please update and provide links related to existing tests, e.g. https://github.com/LD4P/qa_server/tree/master/lib/generators/qa_server/templates/config/authorities/linked_data/scenarios
- [ ] If applicable, please update the relevant tab in https://docs.google.com/spreadsheets/d/1rPvEoP9iYNkxJ0eWC8gXe3ci7e6mhW0da59xkGhadi0/edit?usp=sharing for the particular dataset in question

Thanks.

**Comment by sfolsom** [comment]
The geographic subauth had been in QA for some time now, but for some reason it wasn't added to Sinopia lookup configuration yet. Once https://github.com/LD4P/sinopia_editor/pull/3625 is merged, it will be available to add to any template.
